src/how-to-build-and-run.md

```diff
@@ -84,6 +85,53 @@ compiling `rustc` is done in stages:
   can build the libraries with the stage2 compiler. The result ought
   to be identical to before, unless something has broken.
 
+Thus when running `x.py` you will see output such as:
+
+```
```

Comment:
```suggestion
```txt
```

This will satisfy the CI, I think.

Reply:
Thanks for this.

Reply:
Np. You should be able to apply the suggestion directly from github if that makes things easier for you :)